go.mod

```diff
@@ -19,12 +19,14 @@ require (
 	github.com/go-ole/go-ole v1.2.4 // indirect
 	github.com/go-redis/redis/v8 v8.3.1
 	github.com/gocraft/health v0.0.0-20170925182251-8675af27fef0
+	github.com/golang/mock v1.3.1
```

Comment:
Where are we using it? I don't see a place where its imported 🤔

Reply:
Instead I can see that we import `github.com/stretchr/testify/mock`

Reply:
removed @sredxny. `github.com/stretchr/testify/mock ` is included in `github.com/stretchr/testify `